sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -706,7 +728,9 @@ export class StreamingReceiver extends MessageReceiver {
 
     let sReceiver: StreamingReceiver;
 
-    if (options?._createStreamingReceiver) {
+    if (context.streamingReceiver) {
+      sReceiver = context.streamingReceiver;
+    } else if (options?._createStreamingReceiver) {
```

Comment:
Nit: Can we update the docs for the `create` method with a note that `_createStreamingReceiver` was added to be used in tests and does not fall in the code path of how this package is used? It trips me up every time I follow the code here :)

Reply:
To make it really obvious (because it does seem like we'd use it in production) I renamed it to `_createStreamingReceiverStubForTests`.